src/main/java/seedu/address/storage/Pair.java

```diff
@@ -14,6 +14,15 @@ public class Pair <K, V> implements Serializable {
     private final K key;
     private final V value;
 
+    /**
+     * Fulfils the role of a dummy constructor for JSON file reading compatibility.
+     * I don't know why this is needed. DO NOT USE IN CODE.
+     */
+    public Pair() {
```

Comment:
Similar to my other comment in PR #195. Would it better to use private accessor here? We wouldn't want to use this dummy constructor.

Reply:
Pair Class will not be necessary in the subsequent PRs. Will be removing Pair.java. So closing this as resolved.